scheduler/src/cook/kubernetes/api.clj

```diff
@@ -104,6 +104,17 @@
        :user user}
       instance-uuid (assoc :instance-uuid instance-uuid))))
 
+;; TODO(alexh): should we use another method to determine this? Label?
```

Comment:
I think an annotation would be better than a label, and yes, we should have a better method for this eventually.